res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -97,8 +97,8 @@ PioneerDDJSX.samplerCueGotoAndPlay = false;
 PioneerDDJSX.autoPFL = true;
 
 // If true, new in Mixxx 2.1 introduced library controls will be used,
-// else old playlist controls will be used (default: false).
-PioneerDDJSX.useNewLibraryControls = false;
+// else old playlist controls will be used (default: true).
+PioneerDDJSX.useNewLibraryControls = true;
```

Comment:
Is this option necessary? Can we remove it?

Reply:
Considering you just changed the shifted behavior to act like the unshifted behavior when this option is false, I think the option should be removed.

Reply:
You're right! I removed the option and its If-else dependencies in the library concerned functions.